### DISCUSSION TITLE
"Robot" is derived from a word from what language?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#668] [Galaxy Trivia] [answered]
**Bonus:** What is the approximate number of native speakers of this language?

**Comment by ParanoidUser** [comment]
@valeraqwe, @jvsdo, @Meriem-BM any ideas? 🤐

**Comment by gotthejuicee** [comment, marked as answer]
The word "robot" is derived from the Czech word "robota," which means "forced labor" or "work." The term was introduced into English by Czech writer Karel Čapek in his 1920 play "R.U.R. (Rossum's Universal Robots)." In the play, robots are artificial workers that rebel against their human creators. The term "robot" has since become a common word in English and is used to refer to a wide range of automated machines and devices.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/valeraqwe?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Comment by Meriem-BM** [comment]
The word "robot" comes from the Czech word "robota," which means "forced labor" or "work." The term was coined by Czech writer Karel Čapek in his 1920 play "R.U.R." (Rossum's Universal Robots), which was about artificial workers that rebel against their human creators. The play popularized the term "robot" and it has since become a widely recognized word in many languages to refer to machines that can perform tasks autonomously.
